src/Microsoft.AspNet.JsonPatch/Resources.resx

```diff
@@ -129,9 +132,21 @@
   <data name="InvalidPathForArrayProperty" xml:space="preserve">
     <value>For operation '{0}', the provided path is invalid for array property at path '{1}'.</value>
   </data>
+  <data name="InvalidValueForPath" xml:space="preserve">
+    <value>The provided string '{0}' is an invalid path.</value>
+  </data>
   <data name="InvalidValueForProperty" xml:space="preserve">
     <value>The value '{0}' is invalid for property at path '{1}'.</value>
   </data>
+  <data name="ObjectTypeMustBeJsonPatchDocument" xml:space="preserve">
+    <value>objectType must be of type JsonPatchDocument.</value>
```

Comment:
Any code-names like `objectType` or `JsonPatchDocument` should be passed in as parameter as well.